packages/front/src/lib/components/MultiStepForm/MultiStepForm.svelte

```diff
@@ -5,51 +5,69 @@
 
     export let steps = [];
     export let onSubmit;
+    export let submitLabel = "Submit";
+    export let nextLabel = "Next";
+    export let previousLabel = "Previous";
 
     const controller = new MultiStepFormController(steps, onSubmit);
 
     onDestroy(() => controller.onDestroy());
 
-    const { currentStep, data } = controller;
+    const { currentStep, data, isStepBlocked } = controller;
 </script>
 
-<div class="fr-stepper">
-    <h2 class="fr-stepper__title">
-        <span class="fr-stepper__state">Étape {$currentStep.positionLabel} sur {steps.length}</span>
-        {$currentStep.step.name}
-    </h2>
-    <div class="fr-stepper__steps" data-fr-current-step={$currentStep.positionLabel} data-fr-steps={steps.length} />
-    {#if !$currentStep.isLastStep}
-        <p class="fr-stepper__details">
-            <span class="fr-text--bold">Étape suivante :</span>
-            {$currentStep.step.name || `étape ${$currentStep.positionLabel}`}
-        </p>
+<div class="fr-grid-row">
+    <div class="fr-stepper fr-mb-0">
```

Comment:
```suggestion
    <div class="fr-stepper">
```
Je sais pas d'où ça sortait mais c'était pas beau avec